content/en/docs/tasks/network/validate-dual-stack.md

```diff
@@ -0,0 +1,158 @@
+---
+reviewers:
+- lachie83
+- khenidak
+title: Validate IPv4/IPv6 dual-stack
+content_template: templates/task
+---
+
+{{% capture overview %}}
+This document shares how to validate IPv4/IPv6 dual-stack enabled Kubernetes clusters.
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+* Kubernetes 1.16 or later
+* Provider support for dual-stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)
+* Kubenet network plugin
+* Kube-proxy running in mode IPVS
+* [Dual-stack enabled](/docs/concepts/services-networking/dual-stack/) cluster
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Validate Pod and Node addressing
+
+Validate IPv4/IPv6 Pod CIDRs have been assigned on a dual-stack enabled node by running the following command. There should be a single IPv4 and IPv6 CIDR allocated (replace node name with a valid node from the cluster. In this example the node name is k8s-linuxpool1-34450317-0):
+
+```shell
+kubectl get nodes k8s-linuxpool1-34450317-0 -o go-template --template='{{range .spec.podCIDRs}}{{printf "%s\n" .}}{{end}}'
+10.244.1.0/24
+a00:100::/24
+```
+There should be a single IPv4 and IPv6 CIDR allocated.
+
+Validate that the node has an IPv4 and IPv6 interface detected (replace node name with a valid node from the cluster. In this example the node name is k8s-linuxpool1-34450317-0): 
+```shell
+kubectl get nodes k8s-linuxpool1-34450317-0 -o go-template --template='{{range .status.addresses}}{{printf "%s: %s \n" .type .address}}{{end}}'
+Hostname: k8s-linuxpool1-34450317-0
+InternalIP: 10.240.0.5
+InternalIP: 2001:1234:5678:9abc::5
+```
+
+Validate that a Pod has an IPv4 and IPv6 address assigned. (replace Pod name with a valid Pod on the cluster. In this example the node name is n01-79845568c8-jm9sq)
```

Comment:
nit:
```suggestion
Validate that a Pod has an IPv4 and IPv6 address assigned. (replace the Pod name with a valid Pod in your cluster. In this example the Pod name is `n01-79845568c8-jm9sq`)
```

<ul>
<li>Consider using a friendlier name than <tt>n01-79845568c8-jm9sq</tt></li>
<li>“Validate that a Pod has an IPv4 and IPv6 address assigned” could be a 3<sup>rd</sup> or 4<sup>th</sup> level heading</li></ul>
</ul>